lib/util/config-types.ts

```diff
@@ -301,6 +301,8 @@ export interface ItineraryConfig {
   showLegDurations?: boolean
   showPlanFirstLastButtons?: boolean
   showRouteFares?: boolean
+  /** Whether to show the x minutes late/early in the itinerary body */
```

Comment:
Weird that `AppConfig` has these comments and `ItineraryConfig` doesn't, but I guess having this one be the only item with a comment doesn't really matter?

Reply:
Yeah it's the first of many in this file!